locust/README.md

```diff
@@ -0,0 +1,50 @@
+# Load Test with Locust
+
+This directory contains a Locust script for load testing. 
+
+```
+│   locustfile.py    # gRPC locust client
```

Comment:
Should we only test REST API endpoint instead? I think that might be better. What do you think?

Reply:
oh my bad!